src/BinaryOperation.ts

```diff
@@ -79,9 +80,32 @@ export
         this.dockingPoints["right"] = new DockingPoint(this, this.p.createVector(box.w/2 + (this.s.mBox?.w ?? 50)/4, -(this.s.xBox?.h ?? 50)/2), 1, ["symbol", "differential"], "right");
     }
 
+    /**
+     * It is possible to use the binary minus symbol to form -1.
+     * For nuclear atomic numbers this causes a parsing error because of how
+     * binary operators are currently formatted.
+     *
+      - _right_: right docking point widget to check
+     *
+     */
+    isSolitaryNegativeOne(right: Nullable<Widget>): boolean {
+        if (right?.typeAsString == "Num") {
+            if ((right as Num).getFullText() === "1") {
+                // It would be possible to have
```

Comment:
To have ... ? :P

Reply:
Oh casting my mind back the comment wouldn't be that useful. Just stating the obvious fact that other numbers are indeed possible